app/views/layouts/application.html.erb

```diff
@@ -47,8 +47,10 @@
       <% end %>
     <% end %>
 
-    <main role="main" id="content" class="<%= @content_item.schema_name.dasherize %>" lang="<%= I18n.locale %>">
-      <%= yield %>
+    <main role="main" id="content" class="government-frontend-content" lang="<%= I18n.locale %>">
+      <div id="page" class="<%= content_for?(:page_class) ? yield(:page_class) : '' %>" >
```

Comment:
I don't see any styling associated with `#page` or `.history` - what effect does this change have?

Reply:
No longer needed.